src/HttpWrapper.ts

```diff
@@ -50,7 +50,10 @@ export default class HttpWrapper {
         }
     }
 
-    async sendPromise(url: string, jsonRequestData?: any): Promise<any> {
+    async sendPromise(
+        url: string,
+        jsonRequestData?: unknown
+    ): Promise<unknown> {
```

Comment:
I think this should be a generic function:
```suggestion
    async sendPromise<T>(
        url: string,
        jsonRequestData?: T
    ): Promise<T> {
        if (this.logreq) {
            console.log(
                'sending request:',
                url,
                JSON.stringify(jsonRequestData ?? {}).slice(0, LOGMAXLEN)
            );
        }

        const response = await this.provider.request<T>({
            url,
            method: jsonRequestData != null ? 'POST' : 'GET',
            data: jsonRequestData
        });
        return response.data;
    }
```

Reply:
updated